src/Model/Contact.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace Mailjet\LaravelMailjet\Model;
+
+/**
+* https://dev.mailjet.com/email-api/v3/contactslist-managecontact/
+* Only email is required
+*/
+class Contact
+{
+    const ACTION_ADDFORCE = 'addforce'; # adds the contact and resets the unsub status to false
+    const ACTION_ADDNOFORCE = 'addnoforce'; # adds the contact and does not change the subscription status of the contact
+    const ACTION_REMOVE = 'remove'; # removes the contact from the list
+    const ACTION_UNSUB = 'unsub'; # unsubscribes a contact from the list
+
+    const EMAIL_KEY = 'Email';
+    const NAME_KEY = 'Name';
+    const ACTION_KEY = 'Action';
+    const PROPERTIES_KEY = 'Properties';
+
+    protected $email;
+    protected $name;
+    protected $properties;
+    protected $action;
+
+    public function __construct($email, $name = null, array $properties = [])
```

Comment:
Why not follow same naming convention than other model, which is `optionalProperties`?

If `name` is optional, why put it in the constructor?